scss/_accordion.scss

```diff
@@ -147,3 +147,12 @@
     }
   }
 }
+
+@if $enable-dark-mode {
```

Comment:
I don't have yet an answer but I'm raising the question. Rather than adding a rule for `.accordion-button::after` at then end of the file, wouldn't be more readable to add it where some rules about `.accordion-button` are already defined (in the midldle of everything else)?-like we do for `@if $enable-rounded` elsewhere.

Reply:
Could be! I'm down for whatever makes the most sense for us for maintenance.

Reply:
I resolve this conversation but I keep that in mind for the end of the review, or even after the merge of this PR.